lehmann2008_evaporation/Manifest.toml

```diff
@@ -0,0 +1,274 @@
+# This file is machine-generated - editing it directly is not advised
```

Comment:
I forget if we are supposed to check in Manifest.toml files - it is in the .gitignore file

Reply:
Yes, please add it. One of the points is to enable reproduciability

Reply:
I removed Manifest.toml from the .gitignore :)